### PR TITLE
Codeowner rule fix

### PR DESCRIPTION
**Pull request by k1rk**
so now out of the box we can create alerts with label `code_owner_channel` and they will be routed to appropriate channel.
of course most users have their own routing configuration, but this is nice to have out of the box, and also can be an inspiration for users =)

also updated dependencies =)

### CODE REVIEW

File: charts/victoria-metrics-k8s-stack/Chart.lock [hunk 1]
```diff
@@ -1,15 +1,15 @@
 dependencies:
 - name: victoria-metrics-operator
   repository: https://victoriametrics.github.io/helm-charts
-  version: 0.1.17
+  version: 0.1.19
```
Comment (tenmozes) [inline charts/victoria-metrics-k8s-stack/Chart.lock:4]: Looks good but could you please bump chart version

**Review by tenmozes — CHANGES_REQUESTED** [review]

File: charts/victoria-metrics-k8s-stack/Chart.lock (hunk 1 above)
Comment (k1rk) [reply charts/victoria-metrics-k8s-stack/Chart.lock:4]: what do you mean? 
`victoria-metrics-operator` chart used last available version. 

for this chart i bumped in [a3a3688](https://github.com/VictoriaMetrics/helm-charts/pull/185/commits/a3a3688e96126af65374ec2d7ee714f233386d7a)

**Review by f41gh7 — APPROVED** [review]
LGTM

**Comment by f41gh7** [comment]
Thanks!

**Pull request merged**